src/main/resources/application-local.yml

```
@@ -20,14 +20,4 @@ logging:
-
-cloud:
-  aws:
-    region:
-      static: ${REGION_LOCAL}
-    s3:
-      bucket: ${BUCKET_LOCAL}
-    credentials:
-      access-key: ${AWS_ACCESS_KEY_ID_LOCAL}
-      secret-key: ${AWS_SECRET_ACCESS_KEY_LOCAL}
\ No newline at end of file
+    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)